src/components/ErrorBox.tsx

```diff
@@ -0,0 +1,36 @@
+import classNames from 'classnames';
+import { useEffect } from 'react';
+
+type Props = {
+  errorMessage: string;
+  setErrorMessage: (value: string) => void;
+};
+
+export const ErrorBox: React.FC<Props> = ({
+  errorMessage,
+  setErrorMessage,
+}) => {
+  useEffect(() => {
+    setTimeout(() => {
+      setErrorMessage('');
+    }, 3000);
+  }, [errorMessage]);
```

Comment:
React Hook useEffect has a missing dependency: 'setErrorMessage'. Either include it or remove the dependency array. If 'setErrorMessage' changes too often, find the parent component that defines it and wrap that definition in useCallback